docs/tutorial.md

```diff
@@ -0,0 +1,284 @@
+# BigFlow Tutorial
+
+In this tutorial, we show the complete "Hello, World!" example.
+You will learn how to run the simplest BigFlow [workflow](workflow-and-job.md) on a local machine
+and how to [deploy](deployment.md) it on Cloud Composer.    
+
+## Setup
+This tutorial is based on the [Docs](https://github.com/allegro/bigflow/tree/master/docs) project
+located in the BigFlow repository.
+
+Start from cloning this repository:
+
+```bash
+git clone https://github.com/allegro/bigflow.git
+cd bigflow/docs
+```
+
+Then, [install the BigFlow](../README.md#installing-bigflow) PIP package
+in a fresh `venv` in the `docs` project directory.
+
+Since you have installed the BigFlow PIP package, you can use [BigFlow CLI](cli.md). Test it:
+
+```shell
+bigflow -h
+```
+
+## Run the workflow on a local machine
+
+In BigFlow, your project consists of [workflows](workflow-and-job.md).
+You can run them directly from your local machine or deploy them to Cloud Composer as Airflow DAGs.
+
+Our "Hello, World!" workflow is the simple Python code.
+It's super simple. It consists of two jobs.
+The first one says Hello, and the second one says Goodbye:
+
+[hello_world_workflow.py](examples/cli/hello_world_workflow.py):
+
+```python
+from bigflow.workflow import Workflow
+
+
+class HelloWorldJob:
+    def __init__(self):
+        self.id = 'hello_world'
+
+    def run(self, runtime):
+        print(f'Hello world on {runtime}!')
+
+
+class SayGoodbyeJob:
+    def __init__(self):
+        self.id = 'say_goodbye'
+
+    def run(self, runtime):
+        print(f'Goodbye!')
+
+
+hello_world_workflow = Workflow(
+    workflow_id='hello_world_workflow',
+    definition=[
+        HelloWorldJob(),
+        SayGoodbyeJob()])
+```
+
+The [`bigflow run`](cli.md#running-workflows) command lets you run this workflow directly
+from sources on your local machine (without building and deploying it to Composer). 
+
+### Examples
+
+Run the whole workflow:
+
+```shell
+bigflow run --workflow hello_world_workflow
+```
+
+Output:
+
+```text
+Hello world on 2020-09-10 12:17:52!
+Goodbye!
+```
+
+Run the single job:
+
+```shell
+bigflow run --job hello_world_workflow.say_goodbye
+```
+
+Output:
+
+```text
+Goodbye!
+```
+
+Run the workflow with concrete [runtime](workflow-and-job.md#the-runtime-parameter):
+
+```shell
+bigflow run --workflow hello_world_workflow --runtime '2020-08-01 10:00:00'
+```
+
+Output:
+```text
+Hello world on 2020-08-01 10:00:00!
+Goodbye!
+```
+
+## Selecting environment configuration
```

Comment:
Maybe instead of "Selecting environment configuration" something more generic "Workflow configuration"?

Reply:
done